tests/fixtures/basic-js/.sauce/config.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1alpha
+kind: folio
+sauce:
+  region: us-west-1
+  metadata:
+    name: Playwright Hello World
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+docker:
+  image:
+    name: saucelabs/stt-playwright-jest-node
+    tag: latest
+playwright:
```

Comment:
I know that playwright doesn't have a specific project structure like cypress does, though we'd benefit from having a project folder path that can be set by the user. Just so saucectl knows where to look for files.